docs/features/init.md

```
@@ -0,0 +1,23 @@
+# Init Command
+
+## Summary
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)